Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -3269,7 +3269,6 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
```

Comment:
Basically if you expand this its a part of `FocusDebug` config but somehow `iPhone Distribution` got added along with `iPhone Developer` to it. 
Which was causing cert issues. XCode UI won't even show what the problem was until I checked raw pbxproj file.